src/test/abilities/mycelium_might.test.ts

```diff
@@ -44,62 +44,64 @@ describe("Abilities - Mycelium Might", () => {
   it("will move last in its priority bracket and ignore protective abilities", async () => {
     await game.startBattle([Species.REGIELEKI]);
 
-    const leadIndex = game.scene.getPlayerPokemon()!.getBattlerIndex();
     const enemyPokemon = game.scene.getEnemyPokemon();
+    const playerIndex = game.scene.getPlayerPokemon()?.getBattlerIndex();
     const enemyIndex = enemyPokemon?.getBattlerIndex();
 
     game.move.select(Moves.BABY_DOLL_EYES);
 
-    await game.phaseInterceptor.to(MovePhase, false);
+    await game.phaseInterceptor.run(EnemyCommandPhase);
```

Comment:
```suggestion
    await game.phaseInterceptor.to(TurnStartPhase, false);
```
In case we end up putting something between `EnemyCommandPhase` and `TurnStartPhase` in the future (and it also helps get rid of an import)